packages/x-date-pickers/src/internals/hooks/useField/useField.interfaces.ts

```diff
@@ -79,8 +80,16 @@ export interface FieldSection {
   start: number;
   end: number;
   value: string;
-  emptyValue: string;
+  placeholder: string;
+  /**
+   * Separator used in the input
+   */
   separator: string | null;
+  /**
+   * Separator used to recreate the date from the sections
+   * @default `section.separator`
+   */
+  dateSeparator?: string;
```

Comment:
OK it is not clear then :laughing: 
I render `12 / 12 / 2022` in the input, but I need to parse `12/12/2022` because the real format do not have the new spaces in the separator.
So I'm storing both in state and using one or the other depending on the use case.

Reply:
```tsx
  /**
   * Separator used in the input.
   */
  separator: string | null;
  /**
   * Separator used to recreate the date from the sections.
   * Can be useful when the separator rendered in the input is not the same as the one used for parsing.
   * e.g: ` / ` in the input and `/` in parsing.
   * @default `section.separator`
   */
  dateSeparator?: string;
```

Is that clearer ?